### PR TITLE
[269] Remove search and compare sync

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Search and Compare is dead

### Changes proposed in this pull request
Remove all "syncing"

### Guidance to review
🚢 

### Checklist
- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -3,16 +3,6 @@ authentication:
   # Set this in the env! The below ensures that we are un-authenticatable if we
   # forget to do this in production.
   secret: <%= SecureRandom.base64 %>
-manage_api:
-  # URL of the C# API app (manage-courses-api)
-  base_url: http://localhost:6001
-  # Auth token used between backend and manage-courses-api to authorize requests.
-  secret: banana
```
Comment (defong) [inline config/settings.yml]: https://github.com/DFE-Digital/teacher-training-api/blob/fe1fe8a31a6a6d4b8143520e7bc0a9328f581a3b/azure/template.json#L126-L130

can be removed as well...

Comment (dankmitchell) [reply config/settings.yml]: Yep

File: config/routes.rb [hunk 2]
```diff
@@ -1,117 +1,110 @@
...
-#                                                   api_v2_organisations GET    /api/v2/organisations(.:format)                                                                                                          api/v2/organisations#index
-#                                     api_v2_provider_recruitment_cycles GET    /api/v2/providers/:provider_code/recruitment_cycles(.:format)                                                                            api/v2/recruitment_cycles#index
-#                   sync_courses_with_search_and_compare_api_v2_provider POST   /api/v2/providers/:code/sync_courses_with_search_and_compare(.:format)                                                                   api/v2/providers#sync_courses_with_search_and_compare
-#                    sync_with_search_and_compare_api_v2_provider_course POST   /api/v2/providers/:provider_code/courses/:code/sync_with_search_and_compare(.:format)                                                    api/v2/courses#sync_with_search_and_compare
```
Comment (defong) [inline config/routes.rb]: quick sanity check does frontend call to this has been removed/or need removing?

Comment (dankmitchell) [reply config/routes.rb]: This should be enough - https://github.com/DFE-Digital/publish-teacher-training/pull/958

**Pull request merged**